tests/release/pipelines/fbc_release.go

```diff
@@ -15,186 +15,272 @@ import (
 	"github.com/redhat-appstudio/e2e-tests/pkg/framework"
 	"github.com/redhat-appstudio/e2e-tests/pkg/utils"
 	"github.com/redhat-appstudio/e2e-tests/pkg/utils/tekton"
-	releaseApi "github.com/redhat-appstudio/release-service/api/v1alpha1"
+	//releaseApi "github.com/redhat-appstudio/release-service/api/v1alpha1"
 	tektonutils "github.com/redhat-appstudio/release-service/tekton/utils"
-	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	//"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
 	"k8s.io/apimachinery/pkg/runtime"
 )
 
-var _ = framework.ReleasePipelinesSuiteDescribe("[RHTAPREL-373]fbc happy path e2e-test.", Label("release-pipelines", "fbcHappyPath"), func() {
-	defer GinkgoRecover()
+const (
+	fbcApplicationName              = "fbc-pipelines-aplication"
+	fbcHotfixAppName                = "fbc-hotfix-aplication"
+	fbcComponentName                = "fbc-pipelines-component"
+	fbcHotfixCompName               = "fbc-hotfix-component"
+	fbcReleasePlanName              = "fbc-pipelines-releaseplan"
+	fbcHotfixRPName                 = "fbc-hotfix-releaseplan"
+	fbcReleasePlanAdmissionName     = "fbc-pipelines-releaseplanadmission"
+	fbcHotfixRPAName                = "fbc-hotfix-releaseplanadmission"
+	fbcEnterpriseContractPolicyName = "fbc-pipelines-policy"
+	fbcHotfixECPolicyName           = "fbc-hotfix-policy"
+	fbcServiceAccountName           = "release-service-account"
+	fbcSourceGitUrl                 = "https://github.com/redhat-appstudio-qe/fbc-sample-repo"
+	targetPort                      = 50051
+	relSvcCatalogURL                = "https://github.com/redhat-appstudio/release-service-catalog"
```

Comment:
If https://github.com/redhat-appstudio/e2e-tests/pull/919 merges, this will need to be changed